nav2_collision_monitor/src/range.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2022 Samsung R&D Institute Russia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "nav2_collision_monitor/range.hpp"
+
+#include <math.h>
+#include <cmath>
+#include <functional>
+
+#include "nav2_util/node_utils.hpp"
+
+namespace nav2_collision_monitor
+{
+
+Range::Range(
```

Comment:
The tutorial should be updated that it can support Range sensors too. We make a passive remark about the fact that we use Scans but PC2 is also possible -- we should also mention Range/Sonar/IR